components/connector-service/internal/monitoring/collector/countercollector_test.go

```diff
@@ -0,0 +1,54 @@
+package collector
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestNewCounterCollector(t *testing.T) {
+
+	t.Run("should create and register counter collector", func(t *testing.T) {
```

Comment:
The test promises to verify whether collector was created and registered but asserts check only if result of the function is not nil. "should create counter collection" in more suitable name

Reply:
It also checks if error occured, which would happen if registration failed, but you are right shorter description looks better